components/slack/slack.app.js

```diff
@@ -0,0 +1,45 @@
+const { WebClient } = require('@slack/web-api')
+
+module.exports = {
+  type: "app",
+  app: "slack",
+  methods: {
+    mySlackId() {
+      return this.$auth.oauth_uid
+    },
+    sdk() {
+      return new WebClient(this.$auth.oauth_access_token)
+    },
+    async scopes() {
+      const resp = await this.sdk().auth.test()
+      if (resp.ok) {
+        return resp.response_metadata.scopes
+      } else {
+        console.log("Error getting scopes", resp.error)
```

Comment:
why not throw here?

Reply:
yea generally either errors should be handled better (which they aren't) or just throw so errors properly propagate

Reply:
fixed by throwing resp.error